packages/jsii-calc/lib/compliance.ts

```diff
@@ -20,6 +20,7 @@ import {
   IRandomNumberGenerator,
   Multiply,
 } from './calculator';
+import { SpecialParameter } from './submodule/param';
```

Comment:
This unfortunately creates an illegal dependency cycle in Python:
* `jsii_calc` -> `jsii_calc.submodule.param`
* Loading `jsii_calc.submodule.param` requires loading `jsii_calc.submodule`
* `jsii_calc.submodule` -> `jsii_calc` (it needs `AllTypes`)

Reply:
We can separate this test out into a new submodule similar to the param test.